core/src/components/cat-tooltip/cat-tooltip.scss

```diff
@@ -0,0 +1,24 @@
+:host {
```

Comment:
Time to use the design tokens in this file. start by importing
@use 'variables' as *;
@use 'mixins' as *;